index.html

```diff
@@ -0,0 +1,252 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ayoub Chahir | Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li id="logo"><a href="#" class="logo">My Logo</a></li>
+          <li>
+            <img src="./img/sandwich.svg" alt="" />
+          </li>
+        </ul>
+      </nav>
+      <div class="header-background-shapes">
+        <img src="./img/Ellipse-3.svg" class="ellipse-3" alt="" /><img
+          src="./img/Ellipse-4.svg"
+          class="ellipse-4"
+          alt=""
+        /><img src="./img/Ellipse-5.svg" class="ellipse-5" alt="" /><img
+          src="./img/Ellipse-8.svg"
+          class="ellipse-8"
+          alt=""
+        /><img src="./img/Ellipse-11.svg" class="ellipse-11" alt="" /><img
+          src="./img/Ellipse-12.svg"
+          class="ellipse-12"
+          alt=""
+        /><img src="./img/Ellipse-13.svg" class="ellipse-13" alt="" /><img
+          src="./img/Ellipse-14.svg"
+          class="ellipse-14"
+          alt=""
+        /><img src="./img/Ellipse-15.svg" class="ellipse-15" alt="" /><img
+          src="./img/Rectangle-4.svg"
+          class="rect-4"
+          alt=""
+        /><img src="./img/Rectangle-5.svg" class="rect-5" alt="" /><img
+          src="./img/Rectangle-6.svg"
+          class="rect-6"
+          alt=""
+        /><img src="./img/Rectangle-7.svg" class="rect-7" alt="" />
+      </div>
+      <section class="headline">
+        <h1>
+          I'm Ayoub <br />
+          Glad to see you!
+        </h1>
+        <p class="presentation">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+        <p class="connect">Let's connect</p>
+        <ul class="social-media">
+          <li class="flex-item">
+            <a href=""><img src="./img/twitter.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/Linkedin-icon.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/medium.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/github.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""
+              ><img src="./img/angellist.svg" alt="" class="flex-item"
+            /></a>
+          </li>
+        </ul>
+      </section>
+    </header>
+    <div class="works">
+      <section class="card-works">
+        <img src="./img/Snapshoot-Portfolio.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Tonic</h1>
```

Comment:
Please be sure to change it to a higher heading tag like `h2` or `h3`. A `h1` aim is to act as the web page title of your app and we already have one of those on line 48. Kindly make this change on your multiple project cards 🤓

Reply:
Okay I did

Reply:
Addressed ✔